x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/utility_bar.tsx

```diff
@@ -22,13 +28,15 @@ interface AllRulesUtilityBarProps {
   isAllSelected?: boolean;
   isAutoRefreshOn?: boolean;
   numberSelectedItems: number;
-  onGetBatchItemsPopoverContent?: (closePopover: () => void) => JSX.Element[];
+  onGetBulkItemsPopoverContent?: (closePopover: () => void) => EuiContextMenuPanelDescriptor[];
   onRefresh?: () => void;
   onRefreshSwitch?: (checked: boolean) => void;
   onToggleSelectAll?: () => void;
   paginationTotal: number;
   showBulkActions: boolean;
   hasPagination?: boolean;
+  isBulkActionInProgress?: boolean;
+  disableActions?: boolean;
```

Comment:
Nit: we usually use prefixes like `is`, `has`, `can`, `should` for boolean variables. It helps to understand the meaning of the var when scanning through the code quickly without referring to types. And `disableActions` looks more like an action, rather than a bool state. Let's rename it to something like `isActionsEnabled`.

Reply:
done
I've changed `showBulkActions` as well, as was one of reasons to name `disableActions`